src/box/journal.h

```diff
@@ -272,9 +272,16 @@ static inline int
 journal_write_submit(struct journal_entry *entry)
 {
 	journal_queue_wait();
+	/*
+	 * We cannot account entry after write as in case of bootstrap write
+	 * is synchronous and complete entry synchronously.
+	 */
```

Comment:
I fail to understand what this comment is trying to say.

Reply:
Change to 

```c
        /*
         * We cannot account entry after write. If journal is synchronous
         * the journal_queue_on_complete() is called in write_async().
         */
```